src/backend/tests/unmocked/recruitment.test.ts

```diff
@@ -54,4 +54,25 @@ describe('Recruitment Tests', () => {
       expect(result.dateOfEvent).toEqual(new Date('11/12/24'));
     });
   });
+
+  describe('Get All Milestones', () => {
+    it('Gets all the milestones', async () => {
+      const milestone1 = await RecruitmentServices.createMilestone(
+        await createTestUser(batmanAppAdmin, orgId),
+        'name',
+        'description',
+        new Date('11/11/24'),
+        orgId
+      );
+      const milestone2 = await RecruitmentServices.createMilestone(
+        await createTestUser(batmanAppAdmin, orgId),
```

Comment:
The reason your test is failing is because you are creating a test user for batmanAppAdmin twice. you can only create a user once so either use the same user (create a variable for it outside the milestone and reuse that user. Or create a different user